converters/fromZigbee.js

```diff
@@ -8639,6 +8639,33 @@ const converters = {
             return result;
         },
     },
+    yale_lock_operation_event: {
+        cluster: 'genAlarms',
+        type: ['commandAlarm'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+            if (msg.data.clusterid == 64512) {
+                // We need to read the lock state in case the alarm code is unknown
+                msg.endpoint.read('closuresDoorLock', ['lockState']);
```

Comment:
This needs to be awaited with a try/catch around it. Otherwise z2m will crash with an unhandled promise rejection when the request fails.

Reply:
I have put an await around it, and made it async again in my latest push but, could not find any existing async examples in fromZigbee to use as a template